src/main/java/com/ericsson/ei/EndpointSecurity.java

```diff
@@ -116,23 +117,14 @@ private void addLDAPServersFromList(JSONArray serverList, AuthenticationManagerB
         }
     }
     
-    private boolean checkIfPasswordEncrypted(final String password) {
-        return (password.startsWith("ENC(") && password.endsWith(")"));
-    }
-    
     private String decryptPassword(final String inputEncryptedPassword) throws Exception {
-        TextFormatter textFormatter = new TextFormatter();
-        StandardPBEStringEncryptor encryptor = new StandardPBEStringEncryptor();
-
-        if (jasyptEncryptorPassword.isEmpty()) {
+        if (encryptor.isJasyptPasswordSet()) {
             LOGGER.error("Property -jasypt.encryptor.password need to be set for decrypting LDAP password.");
             throw new AbortExecutionException("Failed to initiate LDAP when password is encrypted. " + 
                                 "Property -jasypt.encryptor.password need to be set for decrypting LDAP password.");
         }
 
-        encryptor.setPassword(jasyptEncryptorPassword);
-
-        String encryptedPassword = textFormatter.removeEncryptionParentheses(inputEncryptedPassword);
+        String encryptedPassword = EncryptionFormatter.removeEncryptionParentheses(inputEncryptedPassword);
         return encryptor.decrypt(encryptedPassword);
```

Comment:
If using these two is common could this be its own method?

Reply:
This can't be done in the EncryptionFormatter since it doesn't have access to the encryptor but could perhaps be done in the Encryptor class. I didn't originally want the Encryptor to be dependant on the formatter class. This thing is done in two places, should I put them in there?